installing/installing_ibm_z/installation-config-parameters-ibm-z.adoc

```diff
@@ -1,7 +1,7 @@
 :_content-type: ASSEMBLY
-[id="installation-config-parameters-ibm-z"]
-= Installation configuration parameters for IBM Z(R) and {linuxoneProductName}
 include::_attributes/common-attributes.adoc[]
+[id="installation-config-parameters-ibm-z"]
+= Installation configuration parameters for IBM Z &#174; and {linuxoneProductName}
```

Comment:
```suggestion
= Installation configuration parameters for IBM Z&#174; and {linuxoneProductName}
```
There shouldn't be a space between the Z and the (R); when I tried this, it wouldn't build.